src/DependencyInjection/CompilerPass/ResourceOwnerMapCompilerPass.php

```diff
@@ -28,7 +28,7 @@ public function process(ContainerBuilder $container)
         $def = $container->getDefinition('hwi_oauth.resource_ownermap_locator');
 
         foreach ($container->getParameter('hwi_oauth.firewall_names') as $firewallId) {
-            $def->addMethodCall('add', [$firewallId, new Reference('hwi_oauth.resource_ownermap.'.$firewallId)]);
+            $def->addMethodCall('set', [$firewallId, new Reference('hwi_oauth.resource_ownermap.'.$firewallId)]);
```

Comment:
What's reason of this?

Reply:
add means you can add something, and here you can not add something - only overwrite if exists.

Reply:
... but its also a sugar. I can revert it - its not significant.

Reply:
After reading more context in okey with this change.